sympy/core/tests/test_numbers.py

```diff
@@ -529,8 +532,10 @@ def teq(a):
 
     # inexact floats (repeating binary = denom not multiple of 2)
     # cannot have precision greater than 15
-    assert Float(.125, 22) == .125
-    assert Float(2.0, 22) == 2
+    assert Float(.125, 22)._prec == 76
+    assert Float(2.0, 22)._prec == 76
+    #assert Float(.125, 22) == .125
```

Comment:
I think commented code should have some reason why it is there (in the comments) and I think linters discourage comments without a space right after the #